akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sbr/SplitBrainResolverIntegrationSpec.scala

```diff
@@ -0,0 +1,465 @@
+/*
+ * Copyright (C) 2015-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sbr
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import com.typesafe.config.ConfigValueFactory
+import org.scalatest.BeforeAndAfterEach
+
+import akka.actor._
+import akka.cluster.Cluster
+import akka.cluster.ClusterSettings.DataCenter
+import akka.cluster.ClusterSettings.DefaultDataCenter
+import akka.cluster.Member
+import akka.cluster.MemberStatus
+import akka.cluster.MultiNodeClusterSpec
+import akka.cluster.sharding.ClusterSharding
+import akka.cluster.sharding.ClusterShardingSettings
+import akka.cluster.singleton.ClusterSingletonManager
+import akka.cluster.singleton.ClusterSingletonManagerSettings
+import akka.pattern.ask
+import akka.remote.testconductor.RoleName
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.testkit.ImplicitSender
+import akka.testkit.LongRunningTest
+import akka.testkit.TestKit
+import akka.testkit.TestProbe
+import akka.util.Timeout
+
+/*
+ * Depends on akka private classes so needs to be in this package
+ */
+object SplitBrainResolverIntegrationSpec extends MultiNodeConfig {
```

Comment:
We should be aware of that the additional SBR tests add 12 minutes to the jenkins multi-node jobs.